build.sbt

```diff
@@ -91,7 +92,7 @@ lazy val kafka =
         "dev.zio"                   %% "zio-streams"             % zioVersion,
         "dev.zio"                   %% "zio-test"                % zioVersion % Test,
         "dev.zio"                   %% "zio-test-sbt"            % zioVersion % Test,
-        "org.apache.kafka"           % "kafka-clients"           % kafkaVersion,
+        "io.conduktor.kafka"         % "kafka-clients"           % kafkaClientsVersion,
```

Comment:
we should probably realse cdk kafka under `org.apache.kafka` group id to be consistent with what we do for other libraries we maintain our own forks for.